SPECS/mesa-libGLU/mesa-libGLU.spec

```
@@ -0,0 +1,130 @@
+Summary:        Mesa libGLU library
+Name:           mesa-libGLU
+Version:        9.0.1
+Release:        4%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://mesa3d.org/
+Source0:        https://mesa.freedesktop.org/archive/glu/glu-%{version}.tar.xz
+Source1:        LICENSE.PTR
+Source2:        make-git-snapshot.sh
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  libtool
+BuildRequires:  mesa-libGL-devel
+#Requires:
+Provides:       libGLU
+
+%description
+Mesa implementation of the standard GLU OpenGL utility API.
+
+%package        devel
+Summary:        Development files for %{name}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Provides:       libGLU-devel
+
+%description    devel
+The %{name}-devel package contains libraries and header files for
+developing applications that use %{name}.
+
+%prep
+%setup -q -n glu-%{?gitdate:%{gitdate}}%{?!gitdate:%{version}}
+cp %{SOURCE1} .
+
+%build
+autoreconf -v -i -f
+%configure --disable-static
+%make_build
+make %{?_smp_mflags}
+
+%install
+%make_install
+
+%post -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
+
+%files
+%license LICENSE.PTR
+%{_libdir}/libGLU.so.1
+%{_libdir}/libGLU.so.1.3.*
+
+%files devel
+%{_includedir}/GL/glu*.h
+%{_libdir}/libGLU.so
+%{_libdir}/libGLU.la
+%{_libdir}/pkgconfig/glu.pc
+
+%post devel -p /sbin/ldconfig
+%postun devel -p /sbin/ldconfig
+%changelog
```

I think running this is not necessary for the `devel` subpackage:

```suggestion
%changelog
```

---

Done.